Assets/_BForBoss/Tests/Tools/TestUtilities.cs

```diff
@@ -8,6 +9,14 @@ public static bool WithinBounds(float firstArg, float secondArg, float bound = 0
         {
             return Math.Abs(firstArg - secondArg) < bound;
         }
+        
+        public static bool WithinBounds(Vector3 firstArg, Vector3 secondArg, float bound = 0.1f)
+        {
+            var x = Math.Abs(firstArg.x - secondArg.x) < bound;
+            var y = Math.Abs(firstArg.y - secondArg.y) < bound;
+            var z = Math.Abs(firstArg.z - secondArg.z) < bound;
+            return x && y && z;
+        }
```

Comment:
Would it make more sense to do `Vector3.Distance(firstArg, secondArg) < bound`? I think if we're comparing each float separately the tolerance gets amplified a lot

Reply:
yeah you're right. I couldn't remember what we would normally use at the time. Will update.